exercises/practice/all-your-base/all-your-base.wat

```diff
@@ -7,7 +7,16 @@
   (global $wrongInputBase i32 (i32.const -2))
   (global $wrongOutputBase i32 (i32.const -3))
 
-  ;; Returns offset and length of resulting u32[] and a return status code
+  ;;
+  ;; Convert an array of digits in inputBase to an array of digits in outputBase
+  ;; @param {i32} arrOffset - base offset of input u32[] array                
+  ;; @param {i32} arrLength - length of the input u32[] array in elements
+  ;; @param {i32} inputBase - base of the input array
+  ;; @param {i32} outputBase - base of the output array
+  ;; @return {i32} - base offset of the output u32[] array
+  ;; @return {i32} - length of the output u32[] array in elements                   
+  ;; @return {i32} - status code (0, -1, -2, -3)                                
+  ;;
```

Comment:
This might be worth doing for _all_ exercises!

Reply:
I concur! I'll add a note to consider doing this as a follow-on.